source/configuration_manual/lazy_expunge_plugin.rst

```diff
@@ -1,36 +1,67 @@
 .. _lazy_expunge_plugin:
 
 ============
-Lazy expunge
+Lazy Expunge
 ============
 
-The idea behind this plugin is that expunged mails and deleted mailboxes stay
-around for a while, so that user can undelete them without assistance from
-sysadmin. The expunged mails won't be counted in user's quota. The plugin
-itself doesn't clean up the expunged messages, you'll have to do it some other
-way (see below).
+The lazy expunge plugin provides a "second-chance" to recover messages that
+would otherwise be deleted from a mailbox by user action.
 
-The plugin is configured by defining namespaces where the mails are moved. You
-can decide if you want the namespaces to be visible to clients, or if you want
-to show them only via some special webmail interface.
+It does this by moving the message to a defined location (either a mailbox, or
+a namespace -- see below for further details) when a user deletes the message
+from a mailbox.
 
+This behavior is useful for a variety of reasons:
+
+#. Protect against misconfigured clients (e.g. POP3 client that deletes all
+   messages)
+#. Protect against accidental deletion (user error)
+#. Archiving
+
+Generally, lazy-expunge is configured so that the expunged mails are not
+counted in the user's quota.  Unless being used for archiving, autoexpunge
+should be used to prune the mailbox to control storage usage.
+
+Configuration
+=============
+
+There are two plugin configuration options:
+
+:``lazy_expunge``: The mailbox/namespace to move messages to when expunged
+:``lazy_expunge_only_last_instance``: If true, only move to expunged storage
+                                      if this is the last copy of the message
+                                      in the user's account
```

Comment:
Could this be a link to the chapter about it?

Reply:
Not sure what you mean by this.

Reply:
so that clicking the "lazy_expunge_only_last_instance" text would bring it to the "Copy only the last instance" chapter.

Reply:
I tried a few things and it didn't work exactly the same. I guess I'll just merge it like this and some day we'll figure out how to make these setting formats more standardized.